pkg/controller/metrics/metrics.go

```diff
@@ -151,7 +151,8 @@ func Add(mgr manager.Manager) error {
 		Client:   mgr.GetClient(),
 		Interval: 2 * time.Minute,
 	}
-	metrics.Registry.MustRegister(newProvisioningUnderwayCollector(mgr.GetClient()))
+	metrics.Registry.MustRegister(newProvisioningUnderwayCollectorWithMinDuration(mgr.GetClient(), 1*time.Hour))
+	metrics.Registry.MustRegister(newProvisioningUnderwayInstallRestartsCollectorWithMinRestarts(mgr.GetClient(), 2))
```

Comment:
My vote would be to start reporting after 1 restart, and this would more closely match when the min restarts start showing up. They're limiting at 3 in OSD so 1 feels a little better for the min.

Reply:
sure, fixed to 1